tools/wpt/run.py

```diff
@@ -265,6 +265,9 @@ def setup_kwargs(self, kwargs):
             kwargs["headless"] = True
             logger.info("Running in headless mode, pass --no-headless to disable")
 
+        if kwargs["browser_channel"] == "nightly" and kwargs["enable_webtransport_h3"] is None:
```

Comment:
Putting this in the frontend seems a bit strange; I guess ideally we want this to be a property of the product. But that's a slightly bigger refactor, so I guess it's fine to do this for now.